functions/venue.js

```diff
@@ -841,11 +858,15 @@ exports.updateMapBackground = functions.https.onCall(async (data, context) => {
     );
   }
 
-  admin
+  const venuesRef = await admin
     .firestore()
     .collection("venues")
-    .doc(venueId)
-    .update({ mapBackgroundImageUrl: data.mapBackgroundImageUrl });
+    .where("slug", "==", data.slug)
+    .where("worldId", "==", data.worldId)
+    .get();
+
+  const venueRef = venuesRef.docs?.[0];
```

Comment:
Node.js is at v10 I think, `?.[]` might break it
```suggestion
  const venueRef = venuesRef.docs && venuesRef.docs[0];
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2548/commits/3d30910bba970e8e3052fe6b53a34775a7b5ed05